datatransfer/impl/graphsync/graphsync_impl_test.go

```diff
@@ -1068,7 +1077,7 @@ type graphsyncTestingData struct {
 	origBytes   []byte
 }
 
-func newGraphsyncTestingData(t *testing.T, ctx context.Context) *graphsyncTestingData {
+func newGraphsyncTestingData(t *testing.T, ctx context.Context) *graphsyncTestingData { // nolint: golint
```

Comment:
what is the linting error we're trying to avoid here?

Reply:
`context.Context should be the first parameter of a function`   it felt weird to put context first for a test that needs a testing.T and it didn't seem like a problem